app/scss/main.scss

```diff
@@ -1,6 +1,6 @@
 body,
 html {
-    font-family: proxima-nova, "Helvetica Neue", Helvetica, Arial, sans-serif;
+    font-family: "proxima-nova", "Helvetica Neue", Helvetica, Arial, sans-serif;
```

Comment:
I think we can include 'proxima-nova' font in our app, so that everyone will have it.